<?xml version='1.0'?>
<sdf version='1.4'>
  <model name="halfSphere">
    <pose>0 0 0 0 0 0</pose>
    <static>false</static>
    <link name="link">
      <inertial>
        <mass>0.07</mass>
        <inertia>
          <ixx>0.000010</ixx>
          <iyy>0.000010</iyy>
          <izz>0.000010</izz>
        </inertia>
      </inertial>
      <collision name="collision">
        <geometry>
          <mesh>
  	        <scale>0.01 0.01 0.01</scale>
            <uri>model://halfSphere/meshes/halfSphere.dae</uri>
          </mesh>
        </geometry>
        <surface>
          <contact>
            <ode>
              <max_vel>0.1</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
        </surface>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
  	        <scale>0.01 0.01 0.01</scale>
            <uri>model://halfSphere/meshes/halfSphere.dae</uri>
          </mesh>
        </geometry>
        <material>
          <script>
            <name>Gazebo/WoodTile</name>
            <uri>file://media/materials/scripts/gazebo.material</uri>
          </script>
          <!--ambient>1 0 0 1</ambient>
          <diffuse>1 0 0 1</diffuse-->
        </material> 
      </visual>  
    </link>    
  </model>
</sdf>
